compiler/rustc_codegen_ssa/src/back/link.rs

```diff
@@ -1238,6 +1238,11 @@ fn exec_linker(
         err.raw_os_error() == Some(ERROR_FILENAME_EXCED_RANGE)
     }
 
+    #[cfg(not(any(unix, windows)))]
+    fn command_line_too_big(_: &io::Error) -> bool {
+        false
```

Comment:
I'm not sure if `true` is more suitable here, since returning `false` means it will never fallback to an `@`-file.

Reply:
Not sure. On one hand defaulting to true could cause linking to fail in some cases. On the other hand it would be a slight performance regression that is likely small enough that no-one will investigate, while failed linking would cause people to notice it and patch rustc to add an implementation of `command_line_too_big` for their platform.

Reply:
Let's keep it `false` for now.